internal/cmd/build-iso.go

```diff
@@ -27,10 +27,10 @@ var BuildISOCmd = cli.Command{
 			Usage:   "The cloud config to embed in the ISO",
 		},
 		&cli.StringFlag{
-			Name:    "name",
+			Name:    "override-name",
 			Aliases: []string{"n"},
-			Value:   KairosDefaultArtifactName,
```

Comment:
I suspect the constant is not needed anymore? (and can be deleted)

Reply:
its used somehwere else. We really need to rework this as soon as its possible :sob: